enterprise/frontend/src/embedding/data-picker/DataSelector/tests/DataSelector.unit.spec.js

```diff
@@ -334,38 +332,6 @@ describe("DataSelector", () => {
     expect(getIcon("chevrondown")).toBeInTheDocument();
   });
 
-  it("should auto-advance past db and schema in field picker", async () => {
```

Comment:
remove tests relating to `fields` I'm not sure why they only fails here since I've removed `fields` reference in #57950 already 🤔